Makefile

```diff
@@ -11,6 +11,7 @@ VERSION ?= 0.8.0
 ifndef PRODUCT_VERSION
 PRODUCT_VERSION := $(shell git describe --tags --dirty --broken)
 endif
+
```

Comment:
@igor-karpukhin after resolving conflicts ... is this PR still necessary if all we're adding is a new line?

Reply:
Looks like this fix was in another PR as well. I'm closing this PR.